docs/installation.rst

```diff
@@ -125,8 +125,21 @@ You'll also need to generate your own ``openssl.ld`` file. For example::
 You should replace the version string on the first line as appropriate for your
 build.
 
+Building cryptography on OS X
+-----------------------------
+
+To install the Xcode command line tools on OS X open a terminal window and run:
```

Comment:
This should probably explain _why_ you want to isntall xcode, perhaps lead with "Building cryptography requires having a compiler installed. [...]"